pkgs/by-name/ja/jan/package.nix

```diff
@@ -17,9 +14,12 @@ appimageTools.wrapType2 {
   inherit pname version src;
 
   extraInstallCommands = ''
-    mv $out/bin/jan-${version} $out/bin/jan
-    install -Dm444 ${appimageContents}/jan.desktop -t $out/share/applications
-    substituteInPlace $out/share/applications/jan.desktop \
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
```

Comment:
Don't re-use `pname`. It creates a binding that goes too far. Imagine if `pname` gets updated, the whole derivation is broken.